### PR TITLE
Make users aware of traefik problems

### PR DESCRIPTION
**Pull request by manuelbuil**
### For Rancher (product) docs only
When contributing to docs, please update the versioned docs. For example, the docs in the v2.6 folder of the `rancher` folder.

Doc versions older than the latest minor version should only be updated to fix inaccuracies or make minor updates as necessary. The majority of new content should be added to the folder for the latest minor version.

### CODE REVIEW

File: content/k3s/latest/en/installation/network-options/_index.md [hunk 1]
```diff
@@ -76,10 +76,10 @@ Dual-stack networking must be configured when the cluster is first created. It c
 
 Dual-stack is supported on k3s v1.21 or above.
 
-To enable dual-stack in k3s, you must provide valid dual-stack `cluster-cidr` and `service-cidr`, and set `disable-network-policy` on all server nodes. Both servers and agents must provide valid dual-stack `node-ip` settings. Node address auto-detection and network policy enforcement are not supported on dual-stack clusters when using the default flannel CNI. Besides, only vxlan backend is supported at the moment. This is an example of a valid configuration:
+To enable dual-stack in k3s, you must provide valid dual-stack `cluster-cidr` and `service-cidr` on all server nodes. Both servers and agents must provide valid dual-stack `node-ip` settings. Note that [Traefik does not correctly handle ingress resources on a dual-stack cluster](https://github.com/traefik/traefik/issues/8406). Besides, only vxlan backend is supported at the moment. This is an example of a valid configuration:
```
Comment (jtravee) [inline content/k3s/latest/en/installation/network-options/_index.md:79]:
```suggestion
To enable dual-stack in K3s, you must provide valid dual-stack `cluster-cidr` and `service-cidr` on all server nodes. Both servers and agents must provide valid dual-stack `node-ip` settings. 

Note that Traefik does not correctly handle ingress resources on a dual-stack cluster. As such, we recommend that users disable Traefik on K3s if they want dual-stack and instead use ingress-nginx. In addition, only vxlan backend is supported currently. The following is an example of a valid configuration:
```

Comment (jtravee) [reply content/k3s/latest/en/installation/network-options/_index.md:79]: We try not to reference GH issues as such in the docs; rather, we try and describe the ongoing issue and workaround for the users. We can add such issues to existing or upcoming release notes if desired/applicable.

File: content/k3s/latest/en/installation/network-options/_index.md [hunk 2]
```diff
@@ -76,10 +76,10 @@ Dual-stack networking must be configured when the cluster is first created. It c
 
 Dual-stack is supported on k3s v1.21 or above.
```
Comment (jtravee) [inline content/k3s/latest/en/installation/network-options/_index.md:77]:
```suggestion
Dual-stack is supported on K3s v1.21 or above.
```

File: content/k3s/latest/en/installation/network-options/_index.md [hunk 3]
```diff
@@ -91,7 +91,7 @@ If you are using a custom cni plugin, i.e. a cni plugin different from flannel,
 IPv6 only setup is supported on k3s v1.22 or above. As in dual-stack operation, IPv6 node addresses cannot be auto-detected; all nodes must have an explicitly configured IPv6 `node-ip`. This is an example of a valid configuration:
```
Comment (jtravee) [inline content/k3s/latest/en/installation/network-options/_index.md:91]:
```suggestion
IPv6 only setup is supported on K3s v1.22 or above. As in dual-stack operation, IPv6 node addresses cannot be auto-detected; all nodes must have an explicitly configured IPv6 `node-ip`. This is an example of a valid configuration:
```

**Review by jtravee — CHANGES_REQUESTED** [review]

**Comment by manuelbuil** [comment]
This is fixed with https://github.com/k3s-io/k3s/pull/5535. It was a code problem on our side

**Pull request closed without merging**